tests/dualtor/test_ipinip.py

```diff
@@ -203,17 +203,18 @@ def setup_uplink(rand_selected_dut, tbinfo):
     # Update the LAG if it has more than one member
     pc_members = mg_facts['minigraph_portchannels'][up_portchannel]['members']
     if len(pc_members) > 1:
-        cmds = [
-            # Update min_links
-            "sonic-db-cli CONFIG_DB hset 'PORTCHANNEL|{}' 'min_links' 1".format(up_portchannel),
-            # Remove 1 portchannel member
-            "config portchannel member del {} {}".format(up_portchannel, pc_members[len(pc_members) - 1]),
-            # Unmask the service
-            "systemctl unmask teamd",
-            # Resart teamd
-            "systemctl restart teamd"
-        ]
+        # Update min_links
+        min_link_cmd = "sonic-db-cli CONFIG_DB hset 'PORTCHANNEL|{}' 'min_links' 1".format(up_portchannel)
+        rand_selected_dut.shell_cmds(cmds=min_link_cmd)
+        # Delete to min_links
+        cmds = "config portchannel member del {} {}".format(up_portchannel, pc_members[len(pc_members) - 1])
         rand_selected_dut.shell_cmds(cmds=cmds)
+        # Ensure delete to complete before restarting service
+        time.sleep(5)
+        # Unmask the service
+        rand_selected_dut.shell_cmds(cmds="systemctl unmask teamd")
+        # Restart teamd
+        rand_selected_dut.shell_cmds(cmds="systemctl restart teamd")
         _wait_portchannel_up(rand_selected_dut, up_portchannel)
```

Comment:
I think, the problem with your solution is that the first command are not guaranteed to be finished before the second command runs.
In my code changes: Line 208 is guaranteed to be run before Line 210 is run.
Hence I made the changes, like this.

Reply:
Team, Please review so that the issue can be closed...

Reply:
@wangxin - could you please review this?

Reply:
Two comments:
1. Use module `shell` instead of `shell_cmds` to run the command of line 208.
2. Use module `shell_cmds` to run rest of the commands in a batch.

How about that?

The `shell_cmds` module is for running a bunch of commands. If just need to run single command, using `shell` module makes more sense.